web/packages/core/src/detect-native-flash.js

```diff
@@ -0,0 +1,62 @@
+import { FLASH_MIMETYPE } from "./ruffle-player.js";
+
+/**
+ * Detect Flash using a handful of APIs.
+ *
+ * If you run this function after loading `plugin-polyfill`, you will only see
+ * the info we added to the current window, which is wrong. Alternatively, if
+ * you want to error-check your plugin falsification, use this function after
+ * installing the falsification.
+ *
+ * This detect is largely similar to the one used by `swfobject` but with
+ * certain browser detects removed.
+ *
+ * Returns if the browser has Flash and the version we detected.
+ */
+export default function detect_flash() {
```

Comment:
What is the idea behind this function? Doesn't seem to be used anywhere. Is it for testing the plugin spoofing?

Reply:
That was something that @kmeisthax added. I think that testing the plugin polyfill was part of the reason & also to do something if the user already has the flash player installed.